src/fireblocks-sdk.ts

```diff
@@ -1150,4 +1153,47 @@ export class FireblocksSDK {
         }
         return await this.apiClient.issueGetRequest(url);
     }
+
+    /**
+     *
+     * @param tokenId
+     */
+    public async getNFT(tokenId: string): Promise<Token> {
+        return await this.apiClient.issueGetRequest(`/v1/nfts/tokens/${tokenId}`);
+    }
+
+    /**
+     *
+     * @param tokenIds comma seperated list of token ids to fetch
+     */
+    public async getNFTs(tokenIds: string): Promise<Token[]> {
+        return await this.apiClient.issueGetRequest(`/v1/nfts/tokens?ids=${tokenIds}`);
+    }
+
+    /**
+     *
+     * @param filter filter results object
+     */
+    public async getOwnedNFTs(filter?: NFTOwnershipFilter): Promise<TokenWithBalance[]> {
+        return await this.apiClient.issueGetRequest(`/v1/nfts/ownership/tokens?${queryString.stringify(filter)}`);
```

Comment:
If we decide to go with an array of strings for ids
Should it be more explicit and not use stringify?

BTW, for which params the `stringify` is needed?